### ISSUE TITLE
Print total coverage in console

### ISSUE THREAD

**Issue by stepango** [Kotlin/kotlinx-kover#229]
After the `koverMergedHtmlReport` task is executed, I only see the link to the report, so to check the actual total coverage value I need to open the report itself. So I want to avoid that. 

Is there any way to print the total coverage value in the console?

**Comment by shanshin** [comment]
Relates #155

**Comment by shanshin** [comment]
@stepango , if I understand correctly, do you want the total percentage of line coverage with the same class-filters used for the HTML report to be displayed in the console?

We planning to add a separate task to output the coverage to the build log, but we would like to understand how to make it convenient: 
- is it necessary to add output formatting
- is it necessary to overload class filters as for other reports
- print only total coverage or it is necessary to print by classes/packages
- which metric should be used: always by lines or additionally need a metric by instructions/branches

**Comment by stepango** [comment]
@shanshin yes that's correct. Back in a days, I used to parse jacoco reports, get total coverage from there and print in console, which is helpful on CI or on local if I just want to check current coverage after the latest changes.

So yeah, for my use case smth as simple as printing total coverage in already existing tasks together with the path to the full report will work.

**Comment by CLOVIS-AI** [comment]
Since the issue about the integration of these metrics with GitLab was merged as duplicate with this one, I'll summarize here what GitLab requires:

• if multiple values are printed, there should be only one per line
• it should be easy to grep for a specific value

The common solution with other tools is to output similar to:
```
Total class coverage: 50.0%
Total branch coverage: 37.5%
...
```

Other than that there are no particular requirements so hopefully this doesn't hinder what you had in mind.

**Comment by dragetd** [comment]
@shanshin I am so happy to see this being integrated! Thank you very much! <3 Looking forward for the next release.

**Comment by shanshin** [comment]
Implemented in `0.7.2`
